### ISSUE TITLE
Set max value of maxItems by calling the command AT+CMGD=?

### ISSUE THREAD

**Issue by 1technophile** [skorokithakis/A6lib#26]
In the example maxItems is defined to 50, it could be better to set it at startup level of the object following the result of AT+CMGD=?

Infos there:
http://www.developershome.com/sms/cmgdCommand.asp

**Comment by skorokithakis** [comment]
Should it be at startup, or right before running any SMS code? We might want to remove the maxItems argument altogether, seems to me? Or, at least, provide the user with a command to get that, so they don't have to run it over and over.

**Comment by 1technophile** [comment]
you are right it is strongly related to sms functions , maybe something like init SMS with the storage as a parameter?

**Comment by skorokithakis** [comment]
Hmm, which storages do you mean?

**Comment by 1technophile** [comment]
storage of SMS, SM, ME..;
http://www.developershome.com/sms/cpmsCommand.asp
